### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by ilyassatlassi**
## Hi, Reviewer 👋👋

In This PR:

- Adding any missing JSX elements to  project
-  Final design  match Zeplin's design
-  Use the CSS rules detailed in Zeplin to style the website.

👌🙌

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -5,23 +5,21 @@ import Categories from './components/Categories';
...
         </div>
       </nav>
-      <div className="footer"><p>Bookstore © 2023</p></div>
       <Routes>
         <Route path="/" element={<Books />} />
         <Route path="/Categories" element={<Categories />} />
       </Routes>
-    </>
+    </div>
   );
```
Comment (BertrandConxy) [inline src/App.js]: - [x] Good job so far 👏  In addition to what you have done, I would suggest that you add the icon on the right side of the nav bar as seen on the design given. Please look at the following comparison 👇 
Your design
image
Given design
image

File: src/components/book.js [hunk 2]
```diff
@@ -12,22 +13,42 @@ function Book(props) {
...
+        <ul className="book-list-item-action-list">
+          <li className="book-list-item-action-list-item"><button type="submit">Comments</button></li>
+          <hr />
+          <li className="book-list-item-action-list-item">
+            <button type="button" onClick={() => dispatch(deleteBooks(id))}>Delete</button>
+          </li>
+          <hr />
+          <li className="book-list-item-action-list-item"><button type="submit">Edit</button></li>
+        </ul>
+
```
Comment (BertrandConxy) [inline src/components/book.js:34]: - [ ] I would suggest that you align well the Update progress button together with the content above it. Please look at the comparison below 👇 
Given design
image
Your design
image

File: src/components/formBook.js [hunk 3]
```diff
@@ -30,9 +30,9 @@ function Form() {
     <div className="book-form">
       <h2>Add Book</h2>
       <form>
-        <input placeholder="Book title" ref={titleVal} />
-        <input placeholder="Book author" ref={authorVal} />
-        <input type="submit" value="Submit" onClick={handleClick} />
+        <input className="input-title" placeholder="Book title" ref={titleVal} />
+        <input className="input-author" placeholder="Book author" ref={authorVal} />
+        <input className="btn" type="submit" value="Submit" onClick={handleClick} />
```
Comment (BertrandConxy) [inline src/components/formBook.js]: - [ ] Please align the Submit button together with the other elements. Please look at the following comparison 👇 
Your design
image
Given design
image

HINT: As you can see it stretches to the end like the above cards

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @ilyassatlassi, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amon-cofie — CHANGES_REQUESTED** [review]
Hi @ilyassatlassi  , 👋 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.




 ## Highlights

🚀 You have a descriptive pull request summary
🚀 You have good commit messages
🚀 You used Gitflow properly
🚀 You have no linter errors


 ## Required Changes ♻️

You have evidently put in a lot of effort to implement the changes that the previous reviewer requested 👍. However, some new bugs may have been implemented. Please observe from the screenshot below: 
image


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @ilyassatlassi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Good UI design :white_check_mark: 
- [x] Application works as expected :white_check_mark: 
- [x] Gitflow is used correctly :white_check_mark:   

## Optional suggestions

- [Optional] Kindly change the title of the project on your readme file, I believe the title of this project ought to be `Bookstore` and not `Math Magician` please rectify this.

Screenshot from 2023-04-08 18-52-46



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**